channels.json

```diff
@@ -37,7 +37,7 @@
     },
     {
       "name": "Tu Huynh",
-      "avatar_url": "",
+      "avatar_url": "https://tuhuynh.com/tu-huynh.jpg",
```

Comment:
Sorry but we do not support loading avatars from an URL yet, could you pls add this file to `public/avatars` folder? 😅

Reply:
Updated 😄